remotespark/sparkkernelbase.py

```diff
@@ -38,6 +39,7 @@ def __init__(self, implementation, implementation_version, language, language_ve
         self.logger = Log(self.client_name)
         self.session_started = False
         self._fatal_error = None
+        self._ipython_display = IpythonDisplay()
```

Comment:
This is private variable, we should add underscore at the beginning to follow the same concept in the code, right?